### PR TITLE
Merge `next` into `master`

### PR DESCRIPTION
**Pull request by seadowg**
We end up with a merge commit (from `master` back into `next`) to make this all work, but I think that's better than rebasing so that we don't change commit SHAs and clobber any useful links between commits and PRs. The most important thing to review is the merge commit itself.

### CODE REVIEW

File: buildSrc/src/main/java/dependencies/Dependencies.kt [hunk 1]
```diff
@@ -21,7 +21,7 @@ object Dependencies {
     const val androidx_multidex = "androidx.multidex:multidex:2.0.1"
     const val androidx_preference_ktx = "androidx.preference:preference-ktx:1.2.0"
     const val androidx_fragment_ktx = "androidx.fragment:fragment-ktx:${Versions.androidx_fragment}"
-    const val android_material = "com.google.android.material:material:1.8.0"
+    const val android_material = "com.google.android.material:material:1.7.0"
```
Comment (seadowg) [inline buildSrc/src/main/java/dependencies/Dependencies.kt:24]: Looks like there are problems using `MaterialCardView` with a Material 3 style and a non-Material 3 theme that we need to solve to upgrade to 1.8.0.

File: collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/GoogleDriveDeprecationTest.kt [hunk 2]
```diff
@@ -56,27 +55,6 @@ class GoogleDriveDeprecationTest {
             .assertText(R.string.google_drive_deprecation_message)
     }
 
-    @Test
-    fun bannerDisappearsAfterSwitchingFromGoogleDriveProjectToOdkServer() {
```
Comment (seadowg) [inline collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/GoogleDriveDeprecationTest.kt:49]: This test is no longer relevant.

Comment (seadowg) [inline collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/GoogleDriveDeprecationTest.kt:49]: This test is no longer relevant.

File: buildSrc/src/main/java/dependencies/Dependencies.kt (hunk 1 above)
Comment (seadowg) [inline buildSrc/src/main/java/dependencies/Dependencies.kt:24]: It looks like there is a problem with `MaterialCardView` using a Material 3 style with a non-Material 3 theme that we need to address to upgrade to 1.8.0

**Review by grzesiek2010 — APPROVED** [review]

**Pull request merged**